### DISCUSSION TITLE
How to connect to SQL using windows Auth?

### DISCUSSION THREAD

**Discussion by vkydhyani** [apache/hop#4654] [General]
is it possible using SQL Native connection ? Can someone please guide me is 2.10.0 SQL native driver can connect to SQL using windows auth.

**Comment by bamaer** [comment]
did you check the [MS SQL documentation](https://hop.apache.org//manual/latest/database/databases/mssqlnative.html#_integrated_authentication_windows_based_authentication)?

**Reply by vkydhyani** [reply]
THANKS.
